### PR TITLE
Update Hyperion documentation to include new effect sensor

### PR DESCRIPTION
**Pull request by belangp**
## Proposed change
Added description of new effect sensor for Hyperion

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102333

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/hyperion.markdown [hunk 1]
```diff
@@ -71,6 +72,10 @@ Please note that only the currently live Hyperion priority can be streamed, and
 streamable sources will actually stream content (e.g., USB Capture Devices will work, but
 static colors will not).
 
+## Sensors
+
+A sensor (Visible Priority) provides the effect currently displayed by the Hyperion server for the selected instance. Attributes of this sensor allows to obtain more details on the nature of the effect as described in the [Hyperion API](https://docs.hyperion-project.org/en/json/ServerInfo.html#priorities)
```
Comment (c0ffeeca7) [inline source/_integrations/hyperion.markdown]:
```suggestion
A sensor (Visible Priority) provides the effect currently displayed by the Hyperion server for the selected instance. Attributes of this sensor provide more details on the nature of the effect. For a detailed description, refer to the [Hyperion API](https://docs.hyperion-project.org/en/json/ServerInfo.html#priorities).
```

**Comment by c0ffeeca7** [comment]
A bit beyond the scope of this PR maybe, but the ha_categories in the header are missing some categories.
image

**Comment by c0ffeeca7** [comment]
Is there a particular reason this PR is targeting the *current* branch?

As the related core PR is on *dev*, it looks like this PR needs to be rebased and target the *next* branch.

**Comment by belangp** [comment]
> Is there a particular reason this PR is targeting the _current_ branch?
> 
> As the related core PR is on _dev_, it looks like this PR needs to be rebased and target the _next_ branch.

@c0ffeeca7 you’re likely right on this. This is my first time contributing to this project and I might have messed that up. Can you advise on how to do this as I don’t want to risk breaking anything?

**Comment by c0ffeeca7** [comment]
1. I would create a new branch off of *next*. 
2. Apply the changes manually again.
3. Create a new PR. In that PR, mention this PR. 
4. Then, close this PR.
It is not the prettiest solution, because this way, you will lose the history, but I don't think that's a big issue in this case. The alternative via git is a bit involved.

**Comment by belangp** [comment]
@c0ffeeca7 I couldn't do another fork, but it let me change the destination of the PR. Is this good enough by any chance?

**Comment by c0ffeeca7** [comment]
> @c0ffeeca7 I couldn't do another fork, but it let me change the destination of the PR. Is this good enough by any chance?

Haha, yes. You got lucky there. :four_leaf_clover: 
This might not work every time. For another time, base it off *next* if it is related to a code PR.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @belangp :+1:

**Pull request merged**